File: decompiled/3.13.1/be/ehealth/businessconnector/therlink/domain/jaxb/Therapeuticlink.java
package be.ehealth.businessconnector.therlink.domain.jaxb;

import be.fgov.ehealth.hubservices.core.v2.TherapeuticLinkType;
import javax.xml.bind.annotation.XmlRootElement;

@XmlRootElement(
   name = "therapeuticlink",
   namespace = "http://www.ehealth.fgov.be/hubservices/core/v2"
)
public class Therapeuticlink extends TherapeuticLinkType {
   private static final long serialVersionUID = 1L;
}

File: src/main/kotlin/org/taktik/connector/business/therlink/domain/jaxb/Therapeuticlink.kt
package org.taktik.connector.business.therlink.domain.jaxb

import be.fgov.ehealth.hubservices.core.v2.TherapeuticLinkType
import javax.xml.bind.annotation.XmlRootElement

@XmlRootElement(name = "therapeuticlink", namespace = "http://www.ehealth.fgov.be/hubservices/core/v2")
class Therapeuticlink : TherapeuticLinkType() {
    companion object {
        private val serialVersionUID = 1L
    }
}
